frontend/src/app/application-forms/validators/number-of-cords-validation.ts

```diff
@@ -0,0 +1,22 @@
+import { FormControl, ValidatorFn } from '@angular/forms';
+
+export function numberOfCordsValidator(num, minNum = -99999): ValidatorFn {
+  return (control: FormControl): { [key: string]: any } => {
+    const val = control.value;
+    if (val) {
+      const phoneNumberRegex = /^(\d+)$/;
+      const valid = phoneNumberRegex.test(val);
+      if (val < minNum) {
```

Comment:
I'm assuming we can remove the phoneregex and valid phoneregex in this function

Reply:
@mtlaney - ah yeah. I changed the REGEX but forgot to change the name. I updated that just now.